cmd/get/matchable_plugin_override.go

```diff
@@ -0,0 +1,112 @@
+package get
+
+import (
+	"context"
+
+	"github.com/flyteorg/flytectl/cmd/config"
+	sconfig "github.com/flyteorg/flytectl/cmd/config/subcommand"
+	pluginoverride "github.com/flyteorg/flytectl/cmd/config/subcommand/plugin_override"
+	cmdCore "github.com/flyteorg/flytectl/cmd/core"
+	"github.com/flyteorg/flyteidl/gen/pb-go/flyteidl/admin"
+)
+
+const (
+	pluginOverrideShort = "Gets matchable resources of plugin override"
+	pluginOverrideLong  = `
+Retrieves plugin overrides for given project and domain combination or additionally with workflow name.
+
+Retrieves plugin overrides for project and domain
+Here the command get plugin override for project flytectldemo and development domain.
+
+::
+
+ flytectl get plugin-override -p flytectldemo -d development 
+
+eg : output from the command
+
+.. code-block:: json
+
+ {
+	"project": "flytectldemo",
+	"domain": "development",
+	"overrides": [{
+		"task_type": "python_task",
+		"plugin_id": ["pluginoverride1", "pluginoverride2"],
+        "missing_plugin_behavior": 0 
```

Comment:
mind commenting on what the enum values represent here?

Reply:
Done.

Reply:
Have added this in yaml . since this is json o/p and comments are supported for JSON format.

If i add comments sphinx doesn't like it . I tried both # and //
```flytectl_get_plugin-override.rst:45: WARNING: Could not lex literal_block as "json". Highlighting skipped.``